draft-frost-rats-eat-collection.md

```diff
@@ -85,6 +85,11 @@ A specific manifestation of such a device is one incorporating the Arm Confident
 
 To support the CCA use case, it is also relevant to consider current attestation technologies which are based on certificate chains (e.g. SPDM, DICE, several key attestation systems). Here also are multiple objects with their own integrity and an internally defined relationship. If attempting to move such a technology to the EAT world, the same challenges apply.
 
+An additional use case beyond the production of tokens from an Attester occurs when using EAT to convey Attestation Results from a Verifier.
+Attestation results may be separated into different sections depending upon what aspects of Appraisal Policy are applied by the Verifier.
+For example, the set of validated evidence claims may form one section, while claims reflecting semantic conclusions drawn by an Appraisal Policy could form another section.
+Given the role of different authorities in concluding result sections, each could have a different signer rather than all results being under a single signature from the Verifier. In this case, a collection can be used to collate all result sections into a single response message.
```

Comment:
I'd add that this enables the collection can be split at a later point in the processing chain if different relying parties need to consume different parts of the Attestation Results.

Reply:
for some reason I don't see this.  Maybe a case of forgotten commit / bad merge?

Reply:
Should be part of [d570a3d] (added the sentence: "Using a collection simplifies operations if individual sections from the collated result sections need to be later dispersed to different Relying Parties.")

Reply:
apologies, it was a case of me looking in the wrong place.